src/components/paradox/results/personality/career/modal/index.js

```diff
@@ -83,19 +83,19 @@ function CareerModal(props) {
                   Career Info
                   <hr className={isSelected("career") ? style.blueDivider : style.grayDivider} />
                 </ModalTab>
-                {clubs && (
+                {clubs?.length > 0 && (
```

Comment:
Is `?` part of the standard now? I just remember using it in coffeescript

Reply:
Yeah the optional chaining operator is standard now! Comes in handy pretty often!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining